samples/interceptors/certs/mg.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this file insertion intentional?

Reply:
The whole dir `samples/interceptors`

Reply:
No. Will be rebasing after removing all of them.